OpenMEEG/cmake/UseACML.cmake

```diff
@@ -0,0 +1,5 @@
+if (USE_ACML)
+    set(ACML_ROOT "/home/papadop/Downloads/acml/gfortran64")
```

Comment:
ACML support has been long ago suspended. I probably kept the code in case we revive this port one day. For now, I do not think it can be activated. But if you can reactivated the support. I'd be happy. That being said, this is also why I support using flens because I think they support ACML, MKL, Atlas, OpenBLAS and maybe more...